frontend/src/scenes/organization/Settings/membersLogic.tsx

```diff
@@ -53,6 +58,18 @@ export const membersLogic = kea<membersLogicType>({
                 return result
             },
         ],
+        filteredMembers: [
+            (s) => [s.members, s.search],
+            (members, search) =>
+                search
+                    ? new Fuse<OrganizationMemberType>(members, {
```

Comment:
Not a big deal but is this perhaps something that should be "memoized"? At the moment every time you search, a new Fuse instance is created, rather than being created newly when the underlying search list (e.g. members) is changed? Micro-optimisation for this code but if it ends up being copy-pasted (or used by me for the Subscription email picker for example...) then it could be worth it.

Reply:
Great point! Fixed this. There might be (are) a few other such cases in the codebase.